docs/behat/basic-usage.rst

```diff
@@ -0,0 +1,513 @@
+Basic Usage
+===========
+
+In our opinion the best way to understand how some stuff works is by showing some examples so this section gather all knowledge form previous chapters.
+Assume that we are implementing managing countries in our system.
+Let the fun begin.
```

Comment:
The best way of understanding how things work in detail is showing and analyzing examples, that is why this section gathers all the knowledge from the previous chapters. 
Let's assume that we are going to implement the functionality of managing countries in our system. Now let us show you the flow.